storages/backends/s3.py

```diff
@@ -691,10 +674,7 @@ def url(self, name, parameters=None, expire=None, http_method=None):
         params["Bucket"] = self.bucket.name
         params["Key"] = name
 
-        connection = (
-            self.connection if self.querystring_auth else self.unsigned_connection
-        )
```

Comment:
hi @jschneier :wave:

I didn't see any point in code where `self.querystring_auth` gets altered after init, and I didn't see any other point in code where `self.unsigned_connection` is used, so this seemed like a safe refactor to me. I didn't look at parent classes though.

Reply:
ah, but the same is not true for `connection`! you're right, will adjust :+1:

Reply:
27fa7c3